pkg/flavor/kubernetes/multiwriter.go

```diff
@@ -0,0 +1,673 @@
+// Copyright 2019 Hewlett Packard Enterprise Development LP
+
+package kubernetes
+
+import (
+	"fmt"
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	log "github.com/hpe-storage/common-host-libs/logger"
+	apps_v1 "k8s.io/api/apps/v1"
+	core_v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/util/mount"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	serviceNamePrefix    = "hpe-multiwriter-svc-"
+	deploymentNamePrefix = "hpe-multiwriter-"
+	pvcNamePrefix        = "hpe-multiwriter-pvc"
+	multiWriterNamespace = "kube-system"
+	nfsImage             = "quay.io/luis_pabon0/ganesha" // TODO evaluate latest nfs-ganesha image
```

Comment:
is this the latest?

Reply:
there are no official ones for this, i will try out building from latest release and verify.